### DISCUSSION TITLE
Include current timestamp in Email subject line

### DISCUSSION THREAD

**Discussion by shafeeqahmd** [grafana/grafana#67092] [Alerting]
Grafana version: v9.4.3
OS: Ubuntu 20.04

Hi

Im trying to include the current timestamp or incident start time in email subject line so that each email is received as separate mail instead of thread/group.

The mail subject template that i tried.
{{ define “email.subject” }}
{{ .Status | toUpper }} - {{ .CommonLabels.alertname }} {{ .Alerts.Firing.StartsAt.Format “Monday, 2 January 2006 at 3:04PM” }}
{{ end }}

Subject that I actually see in mail for the above template: I’m getting only the “FIRING - TestAlert”

“{{ .Alerts.Firing.StartsAt.Format “Monday, 2 January 2006 at 3:04PM” }}” part of the template is just returning empty in actual mail subject alert.

Any help in this would be appreciated. Is there any another option to send each email as separate one?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82740. Feel free to continue the discussion around this there. Thank you!
